packages/builder/test/internal/core/config/config-validation.ts

```diff
@@ -10,93 +10,32 @@ import { expectBuilderError } from "../../../helpers/errors";
 import CfgErrors from "../../../../src/internal/core/config/config-errors";
 
 describe("Config validation", function () {
-  describe("default network config", function () {
-    it("Should fail if the wrong type is used", function () {
-      expectBuilderError(
-        () => validateConfig({ defaultNetwork: 123 }),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
-    });
-  });
-
   describe("paths config", function () {
-    const invalidPathsType = {
-      paths: 123,
-    };
-
-    const invalidCacheType = {
-      paths: {
-        cache: 123,
-      },
-    };
-
-    const invalidArtifactsType = {
-      paths: {
-        artifacts: 123,
-      },
-    };
-
-    const invalidSourcesType = {
-      paths: {
-        sources: 123,
-      },
-    };
-
-    const invalidTestsType = {
-      paths: {
-        tests: 123,
-      },
-    };
-
-    const invalidRootType = {
-      paths: {
-        root: 123,
-      },
-    };
 
-    it("Should fail with invalid types (paths)", function () {
-      expectBuilderError(
-        () => validateConfig(invalidPathsType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
+    const invalidPaths = [
+      {paths: 123},  // invalid path type
+      {paths: {cache: 123}},
+      {paths: {artifacts: 123}},
+      {paths: {sources: 123}},
+      {paths: {tests: 123}},
+      {paths: {root: 123}},
+    ]
 
-      expectBuilderError(
-        () => validateConfig(invalidCacheType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
 
-      expectBuilderError(
-        () => validateConfig(invalidArtifactsType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
-
-      expectBuilderError(
-        () => validateConfig(invalidRootType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
-
-      expectBuilderError(
-        () => validateConfig(invalidSourcesType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
-
-      expectBuilderError(
-        () => validateConfig(invalidTestsType),
-        ERRORS.GENERAL.INVALID_CONFIG
-      );
+    it("Should fail with invalid types (paths)", function () {
```

Comment:
So you prefer to not get a stack trace line in the error but get an error from a loop... Is this better?

Reply:
I can add test-case to the error message. too much copy-paste is not good. Have a look at tabular tests. It basically creates a set of test-cases and then do the execution and checks without copy-pasting.

Reply:
I added a failing config to the message.